tests/backup/backup_test.go

```diff
@@ -2589,29 +2566,7 @@ var _ = Describe("{SharedBackupDelete}", func() {
 		wg.Wait()
 		ctx, err := backup.GetAdminCtxFromSecret()
 		log.FailOnError(err, "Fetching px-central-admin ctx")
-
-		log.Infof("Deleting registered clusters for admin context")
-		err = DeleteCluster(SourceClusterName, orgID, ctx)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cloud cred %s", SourceClusterName))
-		err = DeleteCluster(destinationClusterName, orgID, ctx)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cloud cred %s", destinationClusterName))
-
-		log.Infof("Deleting registered clusters for non-admin context")
-		for _, ctxNonAdmin := range userContexts {
-			err = DeleteCluster(SourceClusterName, orgID, ctxNonAdmin)
-			dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cloud cred %s", SourceClusterName))
-			err = DeleteCluster(destinationClusterName, orgID, ctxNonAdmin)
-			dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cloud cred %s", destinationClusterName))
-		}
-
-		log.Infof("Cleaning up backup location - %s", backupLocationName)
-		err = DeleteBackupLocation(backupLocationName, backupLocationUID, orgID)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Deleting backup location %s", backupLocationName))
-		log.Infof("Cleaning cloud credential")
-		//TODO: Eliminate time.Sleep
-		time.Sleep(time.Minute * 3)
-		err = DeleteCloudCredential(credName, orgID, cloudCredUID)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cloud cred %s", credName))
+		DeleteCloudAccounts(backupLocationMap, credName, cloudCredUID, ctx)
```

Comment:
It looks like log.Infof("Cleaning cloud accounts") before DeleteCloudAccounts is missing.